webview-ui/src/components/chat/TaskHeader.tsx

```diff
@@ -186,25 +184,24 @@ const TaskHeader = ({
 								{!totalCost && <TaskActions item={currentTaskItem} buttonsDisabled={buttonsDisabled} />}
 							</div>
 
-							{doesModelSupportPromptCache &&
```

Comment:
I think the `doesModelSupportPromptCache` check is unnecessary, because the `cacheReads` variable shouldn't be >0 if no cache reads take place.